src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -14,6 +14,9 @@ internal static class CompletionOptions
         public static readonly PerLanguageOption<bool> TriggerOnTyping = new PerLanguageOption<bool>(FeatureName, "TriggerOnTyping", defaultValue: true);
         public static readonly PerLanguageOption<bool> TriggerOnTypingLetters = new PerLanguageOption<bool>(FeatureName, "TriggerOnTypingLetters", defaultValue: true);
 
+        public static readonly PerLanguageOption<int> AddNewLineOnEnterAfterFullyTypedWord =
```

Comment:
This is not the right name for hte option.  It should be something like EnterKeyBehavior